imap/imapd.c

```diff
@@ -2130,6 +2131,13 @@ static void cmdloop(void)
                 cmd_rename(tag.s, arg1.s, arg2.s, havepartition ? arg3.s : 0, /*noisy*/0);
 
                 /* XXX prometheus_increment(CYRUS_IMAP_RENAME_TOTAL); */
+            } else if(!strcmp(cmd.s, "Raclmodseq")) {
```

Comment:
Wouldn't this be better placed in ctl_mboxlist.c, or do we need/want to do this over the wire?

Reply:
It's kind of nice over the wire, because you don't need root access on the server - in the Fastmail code we're calling this from a web session over on a separate server.

It could be a HTTP endpoint instead, but we have lots of other admin-only endpoints in imapd.

Reply:
Reconstruct (for example) doesn't have.  https://datatracker.ietf.org/doc/html/rfc6648 says nah.